fades/main.py

```diff
@@ -173,6 +175,10 @@ def go(argv):
     l.debug("Dependencies from parameters: %s", manual_deps)
     indicated_deps = _merge_deps(ipython_dep, indicated_deps, reqfile_deps, manual_deps)
 
+    # Check for packages updates
```

Comment:
The other option could be check in another thread and show the results at the next run. But is a less explicyt and more complex way. 

If we will check on demand +1